deptofdefense/move.mil
# Update offices data

## Checklist

I have…


## Summary of Changes

This pull request updates the offices seed data and updates the presentation of search results on the Locator Map page.

This PR fixes #138 and fixes #150.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. `git checkout update-offices-data`,
1. set up development dependencies according to `CONTRIBUTING.md`,
1. run `bin/rails rake`,
1. run `bin/rails brakeman`,
1. run `bin/rails rubocop`,
1. run `bin/rails server`,
1. load up http://localhost:3000/resources/locator-maps in your Web browser of choice and try out some searches.

## Screenshots

### Narrow viewport

<img width="447" alt="narrow" src="https://user-images.githubusercontent.com/27780860/33338391-f7966fec-d443-11e7-819b-f8543050bd99.png">

### Wide viewport

<img width="1147" alt="wide" src="https://user-images.githubusercontent.com/27780860/33338392-f7a66758-d443-11e7-8703-476729b6072c.png">
